### DISCUSSION TITLE
running locally: nothing happened

### DISCUSSION THREAD

**Discussion by anlexN** [directus/directus#15575] [Archive]
I have followed [Running Locally](https://docs.directus.io/contributing/running-locally.html) step by step.
Step 1 - 6 is a success.
but Step 7 failed, and nothing happened.
![image](https://user-images.githubusercontent.com/8686739/190177416-39030c8a-3624-4537-a41e-df3deef0b60f.png)

**Comment by br41nslug** [comment]
Can you try running the command in a different terminal? It looks like you're using MinGW (GNU emulator) for windows which may have some weird/unexpected behavior with filesystem interactions (especially watching files). And perhaps only watching the essential packages couldnt hurt 😄 `pnpm -r build` first and then `pnpm -F directus dev` and `pnpm -F @directus/app dev` should be enough in most cases.

**Reply by rijkvanzanten** [reply]
Also I think pnpm might only run 3 things concurrently by default, so `pnpm -r dev` might not start everything as expected

**Comment by anlexN** [comment]
`pnpm -r build` is success done.

`pnpm -F directus dev`
![image](https://user-images.githubusercontent.com/8686739/190296603-716d8f7c-9ead-4d32-996f-8699d9b3d0f7.png)
![image](https://user-images.githubusercontent.com/8686739/190296708-d6e7a526-7f23-4f02-bfbc-e801dde4bd2e.png)
**So what is `8055` meaning?**

`pnpm -F @directus/app dev`
![image](https://user-images.githubusercontent.com/8686739/190297586-d967e917-c6c1-450a-b282-192e56f64db6.png)
![image](https://user-images.githubusercontent.com/8686739/190297634-d63fef61-fc48-43af-b2be-9bc3ee22cd87.png)
![image](https://user-images.githubusercontent.com/8686739/190297678-0f228aae-35ae-414e-bd8a-bdbe2ebf1691.png)
**So why set public base URL to `/admin/`, what is `/admin/admin` meaning?**

**Reply by br41nslug** [reply]
> So what is 8055 meaning?

8055 is the port number, for the running directus api in this case.
> So why set public base URL to /admin/, what is /admin/admin meaning?

I believe this is a side effect from running the App separated from the Api as the api handler the `/admin` routing part normally.

**Reply by anlexN** [reply]
but I can not access localhost:8085, and localhost:8080 directly, why?

**Reply by br41nslug** [reply]
How do you mean not able to access directly? Is it not loading or is it erroring? "localhost:8085" is a typo im presuming.
In your screenshots it looks like it's working and you're connected directly.

**Reply by anlexN** [reply]
so your meaning is: `localhost:8085` can not be accessed in the browser directly, but can be accessed using the app (`localhost:8080`) ?

**Reply by br41nslug** [reply]
`localhost:8085` is a typo right? both `localhost:8055` and `localhost:8080` can both be accessed directly but the first only has the API endpoints while the second only has the App.
